tracing/src/main/java/org/axonframework/extensions/tracing/SpanUtils.java

```diff
@@ -66,7 +67,9 @@
      * @param queryMessage the query to retrieve details from
      * @param queryName    the name provided by the {@link org.axonframework.queryhandling.QueryGateway} caller
      * @return a builder with tags attached
+     * @deprecated
```

Comment:
Isn't a reason desirable for this `@deprecated` ?

Reply:
Nope, you are right, that should be there. Sharp!